internal/internal.go

```diff
@@ -755,3 +759,478 @@ func v2Status(s *ext_type_v3.HttpStatus) *ext_type_v2.HttpStatus {
 		Code: ext_type_v2.StatusCode(s.Code),
 	}
 }
+
+// envoyExtProcGrpcServer represents the ext_proc gRPC server implementation.
```

Comment:
let's split all the `extproc` related code out into it's own subpackage. this file is already really long...